editor/index.js

```diff
@@ -12,6 +19,10 @@ import './assets/stylesheets/main.scss';
 import Layout from './layout';
 import { createReduxStore } from './state';
 
+// Configure moment globally
+moment.locale( settings.l10n.locale );
```

Comment:
Is it independent this way though, if we're assigning defaults to the global instance?